paper-toast.html

```diff
@@ -152,7 +152,11 @@
           noCancelOnOutsideClick: {
             type: Boolean,
             value: true
-          },
+          }
+        },
+
+        listeners: {
+          "transitionend": "_finishTransitioning",
```

Comment:
single quotes please :)
Also, could you name the function `__onTransitionEnd`? (make it private with double _)